classes.h

```diff
@@ -0,0 +1,122 @@
+#pragma once
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+#define PLUS     0
+#define MINUS    1
+#define MULTIPLY 2
+#define DIVIDE   3
+
+class coordinate {
+private:
+    int x;
+    int y;
+
+public:
+    coordinate() : x(0), y(0) {}
+    coordinate(int x, int y) : x(x), y(y) {}
+
+    int getX() { return x; }
+    int getY() { return y; }
+
+    void setX(int value) { x = value; }
+    void setY(int value) { y = value; }
+    void set(int X, int Y) {
+        x = X;
+        y = Y;
+    }
+};
+
+class objects {
+protected:
+    string x_typeOfObject;
+    coordinate* coordinates;
+    int c_length;
+
+public:
+    objects();
+    ~objects();
+    objects(string type, int coordinate_length);
+
+    int getCoordinatesLength() { return c_length; }
+
+    string getType() { return x_typeOfObject; }
+
+    coordinate getCoordinate(int index) { return coordinates[index]; }
+
+    coordinate* getAllCoordinates() { return coordinates; }
+    void setAllCoordinates(coordinate* c, int l) {
+        coordinates = c;
+        c_length = l;
+    }
+
+    void setType(string type) { x_typeOfObject = type; }
+    void setCoordinate(int index, int x, int y) {
+        coordinates[index].setX(x);
+        coordinates[index].setY(y);
+    }
+
+    void outObjects();
+};
+
+class Room
+{
+private:
+    int length;
+    objects** House;
+
+public:
+    Room(int a) : length(a)
```

Comment:
Please, rename variable "a" to more meaningful name.

Reply:
fixed